views/calendar.twig

```diff
@@ -1 +1,34 @@
-{{ message }}
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Technology,Lancaster,Coding,Programming,Meetup">
```

Comment:
no worries. i laughed a little when i saw it too, i just didn't fix it yet either.